package/src/prmon.cpp

```diff
@@ -84,9 +92,29 @@ int ReadProcs(pid_t mother_pid, unsigned long values[4], unsigned long long valu
           fclose(file2);
 	}
 
+        snprintf(stat_buffer,64,"/proc/%llu/stat",(unsigned long long)*it);
+
+        FILE *file3 = fopen(stat_buffer,"r");
+
+        if(file3==0) {
+          openFails.push_back(std::string(stat_buffer));
+        }
+        else {
+          long clock_ticks = sysconf (_SC_CLK_TCK);
+          while(fgets(sbuffer,2048,file3)) {
+            tsbuffer = strchr (sbuffer, ')');
+            if(sscanf(tsbuffer + 2 , "%*c %*d %*d %*d %*d %*d %*u %*u %*u %*u %*u %80llu %80llu %80llu %80llu", &utime, &stime, &cutime, &cstime)) {
```

Comment:
This one I cannot reproduce. Maybe because I'm picking gcc73 via asetup? But I think this should go away once I properly define things as `unsigned long long` above. Please let me know if otherwise.

Reply:
On CentOS7 I didn't get this, but did happen on Ubuntu16.04 with the gcc-7 compiler (which is odd). However, if I added the flag `-Wall` on CentOS7 it was reported.

We should actually get CMake to add that flag in all cases.